tools/cli/lib/api.c

```diff
@@ -1043,3 +1043,224 @@ TDNFCliRefresh(
     return TDNFRefresh(pContext->hTdnf);
 }
 
+static
+uint32_t
+TDNFCliHistoryAlter(
+    PTDNF_CLI_CONTEXT pContext,
+    PTDNF_CMD_ARGS pCmdArgs,
+    PTDNF_HISTORY_ARGS pHistoryArgs
+)
+{
+    uint32_t dwError = 0;
+    PTDNF_SOLVED_PKG_INFO pSolvedPkgInfo = NULL;
+
+    dwError = pContext->pFnHistoryResolve(pContext, pHistoryArgs, &pSolvedPkgInfo);
+    BAIL_ON_CLI_ERROR(dwError);
+
+    if (pHistoryArgs->nCommand == HISTORY_CMD_INIT)
+    {
```

Comment:
?

Reply:
There is nothing to do here for the "init" command. I will add a comment.